test/gtest/group_conv3d_fwd.cpp

```diff
@@ -32,7 +32,8 @@
 #include "get_handle.hpp"
 #include "group_conv3d_fwd.hpp"
 
-struct ConvFwdSolverTest3D : ConvFwdSolverTest<half_float::half>
+// struct ConvFwdSolverTest3D : ConvFwdSolverTest<half_float::half>
+struct ConvFwdSolverTest3D : ConvFwdSolverTest<float>
```

Comment:
[question]
```suggestion
struct ConvFwdSolverTest3D : ConvFwdSolverTest<float>
```
Do we need this comment?